admin_guide/managing_projects.adoc

```diff
@@ -94,3 +102,60 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+[[using-node-selectors]]
+== Using Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. 
+
+[NOTE]
+====
+Labels can be assigned link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+Using node selectors, you can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired set of nodes. For example, if you have a OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods into a specific region. 
```

Comment:
Separate the use of node selectors into two use cases: developer and administrator. The developer can set node selectors on their pod specification (`nodeSelector` entry). The administrator can set a cluster-wide default node selector and override that at the project level. We really need to talk about the interactions at the three levels.

- The project node selector (if any) is the cluster-wide default, potentially overridden by one specified in the project annotation. Thus the project can be less restrictive than the default. In particular, if a project annotation has an empty value (`openshift.io/node-selector: ""`) then that project has no node selector, even if there is a cluster-wide default. This is of particular interest to admins who may want to prescribe a default for users and ignore it for infrastructure projects.
- Whatever the project node selector ends up being, if a user specifies a node selector in their pod spec, this is *merged* with the project node selector, making it more restrictive. The pod node selector cannot end up less restrictive than the project, so the project contents really are restricted to the admin's wishes.